openapi-cli/src/test/java/io/ballerina/openapi/generators/openapi/TestUtils.java

```diff
@@ -72,6 +73,10 @@ public static List<OpenAPIMapperDiagnostic> compareWithGeneratedFile(OASContract
                 String generatedYaml = getStringFromGivenBalFile(tempDir, "payloadV_openapi.yaml");
                 generatedYaml = (generatedYaml.trim()).replaceAll("\\s+", "");
                 expectedYamlContent = (expectedYamlContent.trim()).replaceAll("\\s+", "");
+                if (!(generatedYaml.contains(expectedYamlContent))) {
+                    PrintStream sysout = System.out;
+                    sysout.println("YAML File: " + generatedYaml);
+                }
```

Comment:
I assumed that you are adding this content print for local testing purposes, so shall we remove it?

Reply:
Fixed in e72404dacf39eefdb150129b28476d180161bc78. Thank you for pointing that out.